api/spec/routes/tasks_spec.rb

```diff
@@ -34,4 +34,29 @@ def past_task? task
       task["from_date"] < request_time
     end
   end
+
+  describe "POST /" do
+    subject(:response) { JSON.parse(last_response.body) }
```

Comment:
it would be desirable to test the format of the result, I mean the exact json that returns